src/modules/m_nationalchars.cpp

```diff
@@ -334,7 +334,7 @@ class ModuleNationalChars : public Module
 	}
 
 	/*so Bynets Unreal distribution stuff*/
-	bool loadtables(std::string filename, unsigned char ** tables, unsigned char cnt, char faillimit)
+	bool loadtables(const std::string & filename, unsigned char ** tables, unsigned char cnt, char faillimit)
```

Comment:
```suggestion
	bool loadtables(const std::string& filename, unsigned char** tables, unsigned char cnt, char faillimit)
```

Reply:
@SadieCat, fix it.
Commit info here:
https://github.com/inspircd/inspircd/pull/1879/commits/b4e0c5b6212def02386af0a555445412eed6f1f4